antarest/study/storage/output_service.py

```diff
@@ -0,0 +1,531 @@
+                    return TaskResult(
+                        success=True,
+                        message=f"Study filtered output {study_id}/{output_id} successfully exported",
+                    )
+                except Exception as e:
+                    self.study_service.file_transfer_manager.fail(export_id, str(e))
+                    raise
+
+            task_id = self.study_service.task_service.add_task(
+                export_task,
+                export_name,
+                task_type=TaskType.EXPORT,
+                ref_id=study.id,
+                progress=None,
+                custom_event_messages=None,
+                request_params=params,
+            )
+
+            return FileDownloadTaskDTO(file=export_file_download.to_dto(), task=task_id)
+        else:
+            stopwatch = StopWatch()
+            matrix = StudyDownloader.build(
+                self.study_service.storage_service.get_storage(study).get_raw(study),
+                output_id,
+                data,
+            )
+            stopwatch.log_elapsed(lambda x: logger.info(f"Study {study_id} filtered output {output_id} built in {x}s"))
+            if tmp_export_file is not None:
+                StudyDownloader.export(matrix, filetype, tmp_export_file)
+                stopwatch.log_elapsed(
+                    lambda x: logger.info(f"Study {study_id} filtered output {output_id} exported in {x}s")
+                )
+
+                if filetype == ExportFormat.JSON:
+                    headers = {"Content-Disposition": "inline"}
+                elif filetype == ExportFormat.TAR_GZ:
+                    headers = {"Content-Disposition": f'attachment; filename="output-{output_id}.tar.gz'}
+                elif filetype == ExportFormat.ZIP:
+                    headers = {"Content-Disposition": f'attachment; filename="output-{output_id}.zip'}
+                else:  # pragma: no cover
+                    raise NotImplementedError(f"Export format {filetype} is not supported")
+
+                return FileResponse(tmp_export_file, headers=headers, media_type=filetype)
+
+            else:
+                json_response = to_json(matrix.model_dump(mode="json"))
+                return Response(content=json_response, media_type="application/json")
+
+    def delete_output(self, uuid: str, output_name: str, params: RequestParameters) -> None:
+        """
+        Delete specific output simulation in study
+        Args:
+            uuid: study uuid
+            output_name: output simulation name
+            params: request parameters
+
+        Returns:
+
+        """
+        study = self.study_service.get_study(uuid)
+        assert_permission(params.user, study, StudyPermissionType.WRITE)
+        self.study_service.assert_study_unarchived(study)
+        self.study_service.storage_service.get_storage(study).delete_output(study, output_name)
+        self.study_service.event_bus.push(
```

Comment:
Same here, the event bus must be an explicit dependency of this one, we must not use the one of the study service